lib/tasks/dummy.rake

```diff
@@ -0,0 +1,23 @@
+namespace :dummy do
+  desc 'create dummy data'
+  task create: :environment do
+    u = User.create!(name: 'Pocke')
+
+    cat_with_source = Category.create!(name: 'Vim', description: 'for vim user')
+    source = PostSource.create!(
+      name: 'Vim',
+      user: u,
+      category: cat_with_source,
```

Comment:
__[rubocop]__
* Avoid comma after the last parameter of a method call. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-params-comma)